src/components/common/MyPicker/index.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { string, array } from 'prop-types';
+import { View, TextInput, Text } from 'react-native';
+import ModalPicker from 'react-native-modal-picker'
+
+import styles from './styles';
+
+class MyPicker extends Component {
```

Comment:
This component should be a stateless component since you don't use the state and the life cycle functions

Reply:
You are right, i'll make that change.